src/TypeSpec.Extension/Emitter.Csharp/src/emitter.ts

```diff
@@ -1,203 +1,121 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
-import { createSdkContext } from "@azure-tools/typespec-client-generator-core";
```

Comment:
I was expecting all other *.ts source files can be deleted?  Can we add that and see what it looks like in this pr?

Reply:
all unused *.ts source files are deleted.